### ISSUE TITLE
Using accessibilityIdentifier for UITabBarItem and UIBarButtonItem

### ISSUE THREAD

**Issue by fabb** [calabash/calabash-ios#611]
From iOS 8 on `UITabBarItem` and `UIBarButtonItem` support `accessibilityIdentifier`. It does not show up for calabash though. For `UIBarButtonItem` we still have to set `accessibilityLabel` as a workaround, for `UITabBarItem` we cannot even use this, we have to use the label.
